docs/github/v2/02-activation.md

```diff
@@ -4,71 +4,67 @@ All actions use a Unity installation, which needs to be activated.
 
 Unity differentiates their methods between `personal` and `professional` licenses.
 
+Depending on whether you are using a free or paid version of Unity, you will need to follow the steps for either a [personal license](#personal-license) or a [professional license](#professional-license). 
+
 ## Personal license
 
+Follow these steps if you are using a free version of Unity.
+
 #### Mental model
 
-1. Request activation file for usage on GitHub
+1. Request an activation file for usage on GitHub
 2. Use that file to acquire a license
 3. Set the license as a secret
 
 #### About the activation file
 
 You may use the
 [Unity - Request Activation File](https://github.com/marketplace/actions/unity-request-activation-file)
-action using below instructions.
+action with the below instructions.
 
 The activation file uses machine identifiers and the Unity version number.
-All github virtual machines emit the same hardware ID.
+All GitHub virtual machines emit the same hardware ID.
 You cannot perform this step locally.
 
 Let's go!
 
-### Acquiring an activation file
+#### Acquiring an activation file
 
-Create a file called `.github/workflows/activation.yml` and add a job to it.
+Create a file called `.github/workflows/activation.yml` and add the below workflow definition to it:
 
 ```yaml
 name: Acquire activation file
-on: [push]
+on:
+  workflow_dispatch: {}
 jobs:
   activation:
-    name: Request manual activation file 🔑
+    name: Request manual activation file
```

Comment:
Why remove they icon though? Makes it look modern and recognisable imo.

Reply:
I had copy-pasted from the workflow I tested with (and I generally remove icons just in case there's any encoding issues). I'll add it back, though.